src/Sylius/Bundle/ResourceBundle/Event/ResourceControllerEvent.php

```diff
@@ -130,4 +137,20 @@ public function setErrorCode($errorCode)
     {
         $this->errorCode = $errorCode;
     }
+
+    /**
+     * @param RedirectResponse $redirectRsponse
+     */
+    public function setRedirectResponse(RedirectResponse $redirectRsponse)
```

Comment:
Maybe why should give possibility to set response in general not only `RedirectResponse`.